src/herbpy/action/__init__.py

```diff
@@ -1,2 +1,3 @@
 from grasping import PushGrasp, Grasp
+from rogue import *
```

Comment:
`import *` is evil. Could you explicitly list the functions you wish to import here?